### PR TITLE
add makeBreakpointsHook

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

**Review by smyzin — APPROVED** [review]
Lgtm

File: src/helpers/makeBreakpointsHook.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  return {
+    width,
+    xs: width >= xsSize && width < smSize,
+    sm: width >= smSize && width < mdSize,
+    md: width >= mdSize && width < lgSize,
+    lg: width >= lgSize,
+  }
+}
+
+export function makeBreakpointsHook(calcBreakPoints = calcBreakPointsDefault) {
```
Comment (smyzin) [inline src/helpers/makeBreakpointsHook.ts:18]: Let's make it unnamed

File: src/helpers/makeBreakpointsHook.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  return function () {
+    const [width, setWidth] = useState(window.innerWidth)
+
+    useEffect(() => {
+      function resizer() {
+        setWidth(window.innerWidth)
+      }
+
+      window.addEventListener('resize', resizer)
+      return () => {
```
Comment (smyzin) [inline src/helpers/makeBreakpointsHook.ts]: We can leave `return` without `{}`. What do you think?

File: src/helpers/makeBreakpointsHook.ts (hunk 1 above)
Comment (MixailE) [reply src/helpers/makeBreakpointsHook.ts:18]: We need to use the export name

Comment (smyzin) [reply src/helpers/makeBreakpointsHook.ts:18]: Okay then)

**Pull request merged**